src/asciidoc/reference/packages/withkojiCustomVccSdk.adoc

```diff
@@ -74,18 +78,46 @@ Calling `change` immediately fires an `onUpdate`.
 
 ==== Parameters
 
-* `handler` – Function to handle changes to the custom value.
-* `newValue` – Object containing the current state of the VCC value.
+* `handler` – _Function_, handle changes to the custom value.
+Receives the following property as input:
+** `props` – _Object_, contains the properties of the VCC, see example for full structure.
```

Comment:
It's hard to see the descriptions in the example. However, there's too much info to include the descriptions inline. So, we can try pulling this information into a separate section for **CustomVcc objects**, following the same pattern as in the IAP docs (updated earlier today to make this pattern more standard). The description might be something like this:
```suggestion
** `props` – _Object_, properties of the VCC. See <<_vccprops>>.
```
It can link to a section for VccProps that cross-links back to this method.